blank-page/inc/wpcom-colors-utils.php

```diff
@@ -0,0 +1,192 @@
+<?php
```

Comment:
@scruffian was it determined that we cannot reuse / reference Seedlet's `wpcom-colors-utils.php`?

Reply:
Yeah it will throw an error in Atomic sites